### ISSUE TITLE
🐛[BUG] 使用ProTable组件，搜索查询时，输入框输入 \ 该字符，request中拿到的参数会变成 \\ 

### ISSUE THREAD

**Issue by zhouzhennyu** [ant-design/pro-components#8373]
提问前先看看：

https://github.com/ryanhanwu/How-To-Ask-Questions-The-Smart-Way/blob/main/README-zh_CN.md

### 🐛 bug 描述
搜索查询时，输入框输入 \ 该字符，request中拿到的参数会变成 \\ ，不论输入几个 \\ 该字符，参数都会拿到双倍的 \\ 

![image](https://github.com/ant-design/pro-components/assets/21968050/5d0dbd81-07ee-463b-98c2-afc18c6684c1)

**Comment by fnoopv** [comment]
转义，应该是正常行为?

**Comment by zhouzhennyu** [comment]
老哥，不正常啊，测试提bug了，
正常创建一个名 \ 的内容，筛选 \ 的时候，筛选不到数据了。

**Comment by fnoopv** [comment]
不是 `ProFormText` 的问题，`antd` 的 `Inpu` 组件也是同样的行为，如果认为是bug,应该去 `antd` 提

**Comment by fnoopv** [comment]
事实上转义了才是对的，典型的就是双引号、大于号、小于号等，这些值如果不转义就会造成 `HTML` 标签解析错误，反斜杠同理

**Comment by zhouzhennyu** [comment]
> 事实上转义了才是对的，典型的就是双引号、大于号、小于号等，这些值如果不转义就会造成 `HTML` 标签解析错误，反斜杠同理

antd输入框的原因，但是为啥在创建的时候，如果填写成 \ ,生成的标题不会出现 \\

**Comment by fnoopv** [comment]
因为 `Document` 处理过了,  你在控制台用 `console` 直接打印 `\\`, 输出的也只有一个反斜杠
